xchange-bitmex/src/main/java/org/knowm/xchange/bitmex/service/BitmexAccountService.java

```diff
@@ -0,0 +1,58 @@
+package org.knowm.xchange.bitmex.service;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+
+import org.knowm.xchange.Exchange;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.dto.account.AccountInfo;
+import org.knowm.xchange.dto.account.FundingRecord;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.TradeHistoryParams;
+import org.knowm.xchange.service.trade.params.WithdrawFundsParams;
+
+public class BitmexAccountService extends BitmexAccountServiceRaw implements AccountService {
+
+  /**
+   * Constructor
+   *
+   * @param exchange
+   */
+  public BitmexAccountService(Exchange exchange) {
+
+    super(exchange);
+  }
+
+  @Override
+  public AccountInfo getAccountInfo() {
+    AccountInfo accountInfo = getAccountInfo();
+    return new AccountInfo(accountInfo.getUsername());
+  }
+
+  @Override
+  public String withdrawFunds(Currency currency, BigDecimal amount, String address) throws IOException {
+    return null;
+  }
+
+  @Override
+  public String withdrawFunds(WithdrawFundsParams params) throws IOException {
+    return null;
+  }
+
+  @Override
+  public String requestDepositAddress(Currency currency, String... args) throws IOException {
+    return null;
+  }
+
+  @Override
+  public TradeHistoryParams createFundingHistoryParams() {
+    return null;
+  }
+
+  @Override
+  public List<FundingRecord> getFundingHistory(TradeHistoryParams params) throws IOException {
+    return null;
```

Comment:
Instead of returning null for all these, throw a NotYetImplementedException or NotAvailableFromExchangeException

Reply:
done, will do this for all exchanges going forwards.